externs/shaka/player.js

```diff
@@ -1017,6 +1017,22 @@ shaka.extern.ManifestConfiguration;
 shaka.extern.StreamingConfiguration;
 
 
+/**
+ * @typedef {{
+ *   sourceBufferExtraFeatures: string
+ * }}
+ *
+ * @description
+ *   Media source configuration.
+ *
+ * @property {string} sourceBufferExtraFeatures
+ *   Some platforms may need to pass features when initializing the
+ *   sourceBuffer.
```

Comment:
I think it needs to be explained that this string is ultimately appended to MIME types in addSourceBuffer().  I don't think I would know what this is for or what effect it would have from the current description alone.

Reply:
Done!

Reply:
@joeyparrish can you review again? Thanks!

Reply:
Looks good to me.  Thanks!